<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>222</width>
     <height>119</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Введите колличество строк:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_rows">
        <property name="maxLength">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_random">
      <property name="text">
       <string>Заполнить случайными значениями</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_clear">
      <property name="text">
       <string>Очистить таблицу</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>891</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Предупреждение:  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="warning">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Ошибка:                </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="error">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Время выполнения:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="time">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_min">
            <property name="text">
             <string>Найти минимальное значение </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="output_min">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_max">
            <property name="text">
             <string>Найти максимальное значение</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="output_max">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_avg">
            <property name="text">
             <string>Посчитать среднее значение   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="output_avg">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Строки:         </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="output_find">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Колличество:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="output_count">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Метод:           </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="output_method">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_sort">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Отсортировать таблицу</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>95</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="plus">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="minus">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>300</number>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Числа:</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>0</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>159</width>
     <height>153</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Выберите тип сортировки:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="button_bubble">
      <property name="text">
       <string>Пузырьком</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="button_fast">
      <property name="text">
       <string>Быстрая</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="button_comb">
      <property name="text">
       <string>Расчёска</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="button_gnome">
      <property name="text">
       <string>Гномья</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="button_monkey">
      <property name="text">
       <string>Обезьянья</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Введите искомое число:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_find"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="pushButton_find">
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>Удалить дубликаты</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Строк удалено:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="delete_count">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
